api/api.js

```diff
@@ -116,4 +115,46 @@ app.post('/registerStudent', (req, res) => {
         }
     });
 });
+
+app.patch('/student', (req, res) => {
```

Comment:
What does app.patch do?

Reply:
PATCH is an HTTP method that's very similar to PUT, but it signifies updating a resource instead of creating a new one